pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -256,6 +318,29 @@ func getErrorAsStatus(err error) *framework.Status {
 	return nil
 }
 
+// isPluginMigratedToCSIOnNode checks if an in-tree plugin has been migrated to a CSI driver on the node.
```

Comment:
Thanks, I think this function could also move to the shared library.

Reply:
Ah, yes. Added a new function in `volume/csimigration`